src/pages/Playlists.js

```diff
@@ -1,12 +1,42 @@
+import { useState, useEffect } from "react";
+
 export default function Playlists() {
+  const [playlists, setPlaylists] = useState([]);
+
+  const hash = window.location.hash.substring(1);
```

Comment:
where is the benefit here using window everytime?

Reply:
I think you do have to write window. for it to work or not?